common/app/routes/Settings/Social-Settings.jsx

```diff
@@ -22,6 +22,7 @@ export default function SocialSettings({
     <Button
       block={ true }
       bsSize='lg'
+      bsStyle='primary'
```

Comment:
Looking at this now, I don't understand your reasoning for adding this. None of the other social buttons have a primary style. What is missing is the active state. Maybe you where referring to that? In that case the change would be to add `active={ isGithubCool }`